sherpa/ui/utils.py

```diff
@@ -6850,6 +6840,26 @@ def delete_model(self, id=None):
         self._sources.pop(id, None)
 
     def _check_model(self, model):
+        """Return a Model instance.
+
+        Converts a string to the model instance if necessary.
+
+        Parameters
+        ----------
+        model : str or a sherpa.models.model.Model object
```

Comment:
For rendering in sphinx (if this method is even shown there, but even if not now, maybe we'll at it in the future)
```suggestion
        model : `str` or a `sherpa.models.model.Model` object
```

Reply:
I've given up trying to work out when symbols get automatically annotated or not and just ignored it completely!

Reply:
For reference, this is not referenced / made visible in the RTD docs